pkg/piecestore/psserver/psdb/psdb_test.go

```diff
@@ -310,10 +311,37 @@ func TestHappyPath(t *testing.T) {
 		}
 	})
 
+	t.Run("DeleteObsolete", func(t *testing.T) {
```

Comment:
This should use the actual structure of the production storage node... not some random stuff.

Reply:
can you be little bit more specific. I am bit lost. I created a dummy list of directory with 2 char lengths, similar to what it would like at the production and tested the DeleteObsolete().

Reply:
Egon wanted to say that it cannot be dummy list but directory structure from real storage node. You need to find somewhere (maybe your raspberry pi storage node) such storage node and emulate in test real dir structure.